README.md

```diff
@@ -1,74 +1,270 @@
-* `-v, --verbose` - output debug messages.
-
-### Credentials
-
-Kinvey credentials can be provided as command line options or as environment variables.
-
-#### Command line options
-Run `kinvey config` and the CLI will prompt for Kinvey credentials if they are not already set as environment variables.
-
-You can also create a one-time session using the `-e` and `-p` (and optionally `--host`) flags. Set these parameters explicitly in your calls if you are unable to init with `kinvey config [host]`. It's important to note that the `-e` and `-p` flags take precedence over environment variables. Example:
+
+    kinvey `<command>` [args] [options]
+
+## Commands
+
+`init`
+
+   Prompts you to provide account credentials, host, and profile name.
+
+`profile create <name> [profile information]`
+
+   Creates a profile with the specified name. You can specify the profile information either on the command line as arguments or as environment variables. In the presence of command line argument, any values in the environment variables are ignored.
+
+`profile list`
+
+   Lists all existing profiles. Profiles are saved under the user home.
+
+`profile show [name]`
+
+   Shows detailed information about the specified profile. If you omit the profile name, information about the active profile is shown.
+
+`profile use <name>`
+
+   Sets an active profile.
+    
+`profile delete <name>`
+
+   Deletes the specified profile.
+
+`flex init`
+
+   Configures Kinvey CLI to work with a specific Flex Service. This command is designed to be executed in a Node.js project directory where it creates a `.kinvey` configuration file. Each successive execution in the same directory overwrites the configuration file.
+
+`flex deploy`
+
+   Deploys the current project to the Kinvey FlexService Runtime.
+
+`flex job [id]`
+
+   Shows the job status of a deploy/recycle command. If you don't specify an `id`, the command returns the status of the most recent `flex deploy` or `flex recycle` command.
+
+`flex status`
+
+   Displays the health of the current (the one you initiated last) Flex Service. To get the status of a different service, specify its service ID using the `--serviceId` option.
+
+`flex list`
+
+  Lists all Flex Services for a domain (app or organization), excluding external Flex Services. Specify domain using `--domain` and then an app or organization ID using `--id`. If you skip the domain option, the command lists the services inside the domain you've configured as part of running `flex init`.
+
+`flex logs`
+
+   Retrieves and displays Flex Services logs. Logs calls return 100 entries by default and can return up to 2,000 entries. Logs are displayed in the following format: `<runtime id> <timestamp> - <message>`. Run `kinvey flex logs -h` for advanced `logs` command usage details, including returning more than the default number of entries, specifying a page number, or filtering by timestamp. Log for external Flex Services are not returned.
```

Comment:
'Log for external Flex Services are not returned.'
Plural for log

Reply:
👍